pom.xml

```diff
@@ -444,7 +444,7 @@
         <version.io.smallrye.smallrye-opentelemetry>2.6.0</version.io.smallrye.smallrye-opentelemetry>
         <version.io.smallrye.smallrye-reactive-messaging>4.24.0</version.io.smallrye.smallrye-reactive-messaging>
         <version.io.undertow.jastow>2.2.8.Final</version.io.undertow.jastow>
-        <version.io.vertx.vertx>4.5.9</version.io.vertx.vertx>
+        <version.io.vertx.vertx>4.5.10</version.io.vertx.vertx>
```

Comment:
I filed https://issues.redhat.com/browse/WFLY-19730 and will link this to it and update the PR title so the JIRA appears in the git log. If you need to update this PR for another reason, please amend the commit message.

Also, please add the diff or rn stuff to the WFLY-19730 description. If it resolves any CVEs please change the JIRA title to note that.